doc/source/ndl.rst

```diff
@@ -1,8 +1,30 @@
+Naive Discriminative Learning
+=============================
 
-.. _comparison_of_algorithms:
+Terminology
+-----------
+
+Before explaining Naive Discriminative Learning (NDL) in detail, we want to
+give you a brief overview over important notions:
+
+cue :
+    A cue is something that gives a hint on something else. The something else
```

Comment:
replace *something* with stimulus?

Reply:
I opened an issue for that (#133) to resolve it later